ros_buildfarm/config/loader.py

```diff
@@ -61,6 +61,8 @@ def load_url(url, retry=2, retry_period=1, timeout=10, skip_decode=False):
                 url, retry=retry - 1, retry_period=retry_period,
                 timeout=timeout)
         raise URLError(str(e) + ' (%s)' % url)
+    except OSError as e:
+        raise OSError(str(e) + ' (%s)' % url)
```

Comment:
If a subclass is raised this changes the exception type. Try something like in line 55 instead.